mantidproject/mantid
# Set theta sign axes for D17 and Figaro

This PR adds the `theta-sign` parameters to the IDFs of D17 and Figaro.

**To test:**

Check that the instruments load (`LoadEmptyInstrument`) and that the detectors at workspace indices 0 and 250 have different signs: 
```python
ws.detectorSignedTwoTheta(ws.getDetector(0))
```

Fixes #22391. 

**Release Notes** 

No-one will notice. Doesn't have to be in the release notes.

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
